google/cloud/functions/internal/parse_options_test.cc

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/functions/internal/parse_options.h"
+#include <gmock/gmock.h>
+#include <cstdlib>
+
+namespace google::cloud::functions_internal {
+inline namespace FUNCTIONS_FRAMEWORK_CPP_NS {
+namespace {
+
+TEST(WrapRequestTest, Help) {
+  char const* argv[] = {
+      "unused",
+      "--help",
+  };
+  int argc = sizeof(argv) / sizeof(argv[0]);
+  auto const vm = ParseOptions(argc, argv);
```

Comment:
Test whether `ParseOptions()` can deal with `argc == 0`?

Reply:
It couldn't, or rather, it throws an exception,changed some code and and add a new test.